assets/neuIP-3/token-metadata.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/neume-network/neuIPs/blob/main/assets/neuIP-3/token-metadata.schema.json",
+  "title": "Token Metadata",
+  "description": "NFT token metadata for NewSong event",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "An identifier for the song, preferably in the form of <artist> — <title>.",
```

Comment:
honestly I'm against this artist - title formatting. It's amateurish. We have JSON and we must use it properly despite some early attempts at making stuff work on Opensea

Reply:
Yeah, I know it sucks, but that is the what people are doing. As an engineer, it's somewhat appalling and I would never have designed something like this in a greenfield project. In a live system participants often find clever ways to get the system to do things it wasn't designed for. I think it's fair to capture current usage as a starting point. Then offer better options on top.

Reply:
An alternative here would be to support this as a fallback but include separate "artist" and "title" properties as the "correct" way to do it.

Another is to take the name as-is to be the track title and creator/minter address (or ens) as the artist. This is the approach used by https://neosound.xyz.

Reply:
i'll resolve this for now given the more advanced reasoning in the comments down below.